solr/core/src/java/org/apache/solr/security/AuthenticationPlugin.java

```diff
@@ -95,11 +95,11 @@ public final boolean authenticate(HttpServletRequest request, HttpServletRespons
     }
   }
 
-  HttpServletRequest wrapWithPrincipal(HttpServletRequest request, Principal principal) {
+  protected HttpServletRequest wrapWithPrincipal(HttpServletRequest request, Principal principal) {
```

Comment:
Example of base-class method that required same-package for plugins. With this change the auth plugins can be any package.